proguard/src/main/java/com/flurry/proguard/UploadMapping.java

```diff
@@ -106,60 +115,70 @@ public static Properties parseConfigFile(String filePath) {
     }
 
     /**
-     * Tar a ProGuard file and send it to Flurry's crash service
+     * Tar a ProGuard/Native mapping file and send it to Flurry's crash service
      *
      * @param apiKey the API key for the project being built
      * @param uuid the uuid for this build
-     * @param pathToFile the path to the ProGuard mapping.txt file
+     * @param paths the paths to the ProGuard/Native mapping.txt files
      * @param token the auth token for API calls
      * @param timeout the amount of time to wait for the upload to be processed (in ms)
      */
-    public static void uploadFile(String apiKey, String uuid, String pathToFile, String token, int timeout) {
-        File file = new File(pathToFile);
-        if (file.isDirectory()) {
-            failWithError("{} is a directory. Please provide the path to mapping.txt", pathToFile);
-        }
+    public static void uploadFile(String apiKey, String uuid, String[] paths, String token, int timeout,
```

Comment:
We should probably rename this to `uploadFiles`

Reply:
And maybe we should pass an array of uuids as well.

Reply:
Why should we pass an array of UUIDs? For native SO files we can generate a UUID right before uploading them.